go/otgrpc/errors_test.go

```diff
@@ -0,0 +1,57 @@
+package otgrpc
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/opentracing/opentracing-go/mocktracer"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	firstCode = codes.OK
+	lastCode  = codes.DataLoss
```

Comment:
I couldn't find a better way to iterate through all the GRPC error codes, there doesn't seem to be a native way to do it.